test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -30,13 +30,10 @@
 from . import get_efa_test_instance_type
 
 from .utils import get_ecr_registry, NightlyFeatureLabel, is_nightly_context
-from .integration import (
-    get_framework_and_version_from_tag,
-    get_cuda_version_from_tag,
-    get_processor_from_image_uri,
-)
 from .utils.image_utils import build_base_image, are_fixture_labels_enabled
 
+from test.test_utils import get_cuda_version_from_tag, get_framework_and_version_from_tag
```

Comment:
Did we run SM local tests on the PR? I would suggest running a test. One thing that can be observed in this SM test suite is that we do not use anything from the generic `test_utils` file i.e. `test.test_utils`. 

SM local tests have a different execution style in which we make a tar of the SM test suite: https://github.com/aws/deep-learning-containers/blob/master/test/testrunner.py#L62-L70.

Thus, we might want to test and proceed accordingly.

Reply:
Forgot about that. Change reverted.